lib/hyrax/engine.rb

```diff
@@ -32,10 +32,22 @@ class Engine < ::Rails::Engine
     end
 
     config.after_initialize do
-      begin
+      can_connect = begin
+        ActiveRecord::Base.connection
+        true
+      rescue StandardError
```

Comment:
Can we use `ActiveRecord::ConnectionNotEstablished`, `ActiveRecord::NoDatabaseError` here?

Reply:
Nope. If using Postgres, the error raised (and not trapped by ActiveRecord) is `PG::ConnectionBad`. I tried but couldn't identify a good error or common ancestor to trap aside from `StandardError`.

Reply:
Done.